tests/test-cases/Group0-Bugs/7137.robot

```diff
@@ -0,0 +1,41 @@
+# Copyright 2017 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+*** Settings ***
+Documentation  Test 6515
+Resource  ../../resources/Util.robot
+Suite Setup  Install VIC Appliance To Test Server
+Suite Teardown  Cleanup VIC Appliance On Test Server
+
+
+*** Test Cases ***
+Check for die events when forcing update via state refresh
+    # basic confirmation of function
+    ${rc}  ${id}=  Run And Return Rc And Output  docker %{VCH-PARAMS} pull ${busybox}
+    Should Be Equal As Integers  ${rc}  0
+
+    ${rc}  ${since}=  Run And Return Rc And Output  docker info --format '{{json .SystemTime}}'
+    ${name}=  Generate Random String  15
+    ${rc}  ${id}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run --name ${name} -d ${busybox} sleep 3
+    Should Be Equal As Integers  ${rc}  0
+    ${shortid}=  Get container shortID  ${id}
+
+    # tight loop on inspect
```

Comment:
If this is just to inspect the container above, it is only alive for 3 seconds, so something like this should work:
Run      for i in $(seq 1 5); do docker %{VCH-PARAMS} inspect ${id}; done